reporter/html.js

```diff
@@ -707,8 +710,18 @@ QUnit.testDone(function( details ) {
 		}
 	}
 
+	if (config.collapse !== collapseCheck) {
+		collapse = config.collapse;
+	}
+
 	if ( bad === 0 ) {
 		addClass( assertList, "qunit-collapsed" );
+	} else {
+		if (( collapse === true ) && (collapseFirst === 0 )) {
+			collapseFirst = 1;
```

Comment:
as the class name used below, it would be good to rename this variable to `collapsed`. And you can use booleans.

doing so everything can be simplified:

```js
if ( config.collapse && collapseFirst ) {
```